lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -621,28 +621,29 @@ sub upgrade_info {
     my $dbinfo = $database->get_info();
     my $upgrade_type = "$dbinfo->{appname}/$dbinfo->{version}";
 
+    $request->{lsmb_info} = $database->stats();
 
     if (applicable_for_upgrade('default_ar', $upgrade_type)) {
-    @{$request->{ar_accounts}} = _get_linked_accounts($request, 'AR');
-    unshift @{$request->{ar_accounts}}, {}
-            unless scalar(@{$request->{ar_accounts}}) == 1;
+        @{$request->{ar_accounts}} = _get_linked_accounts($request, 'AR');
```

Comment:
Chris (and others) expressed that we should separate indentation changes from functional changes. From what I can tell, I'm looking at a single commit with both functional and whitespace changes?

Reply:
On this only 1 line, yes. May I remind everyone that adding `?w=1` on the url removes whitespace changes and allows to focus on the issue being fixed

Reply:
well, in https://github.com/ledgersmb/LedgerSMB/pull/3212/commits/b61a26300445dd386e502f0ad454c01d0adf2c06 (a view on 1 commit), I see 1 line added to setup.pm and many lines changing indent.

The problem is that "the future" has to go and try to see why the whitespace changes get attributed to a change which says "AR/AP detection" in 'blame' output.